src/Datasource/Query.php

```diff
@@ -211,40 +507,18 @@ public function setWebservice(WebserviceInterface $webservice)
      *
      * @return \Muffin\Webservice\Webservice\WebserviceInterface
      */
-    public function getWebservice()
+    public function getWebservice(): WebserviceInterface
     {
         return $this->_webservice;
     }
 
-    /**
```

Comment:
Please move back this function to it's original location and avoid unnecessary diff changes

Reply:
Moved